sdk/tables/azure-data-tables/src/main/java/module-info.java

```diff
@@ -7,6 +7,7 @@
     // public API surface area
     exports com.azure.data.tables;
     exports com.azure.data.tables.models;
+    exports com.azure.data.tables.sas to com.azure.core;
```

Comment:
Why is this exporting only to `core`? If this is a non-implementation package, it should be exported to all.

Reply:
I didn't notice my IDE did this. I will export it to all just like the other packages.